lib/index.js

```diff
@@ -55,6 +56,55 @@ exports.plugin = {
             server.ext('onPreStart', internals.initialize);
             server.ext('onPostStop', internals.stop);
 
+            // Expose an hpal command for schwifty-migration
+            server.expose('commands', {
+                'migrate:diff': (initializedServer, args, rootDir, ctx) => {
+
+                    const SchwiftyMigrateDiff = require('./schwifty-migrate-diff').get();
+
+                    if (!SchwiftyMigrateDiff) {
+                        throw new ctx.DisplayError('To use this command please first install `npm install --save-dev schwifty-migrate-diff`');
+                    }
+
+                    const [mode, migrationName, argsMigrationsDir] = args;
```

Comment:
Should these be specified with flags?  Which are required/optional?

Reply:
These are all optional, and they are validated in schwifty-migrate-diff